Makefile

```
@@ -197,20 +197,11 @@ check: test
 # Remove files generated by running coverage, put the non-instrumented lib back
 # in place
 coverage-clean:
-	if [ -d lib_ ]; then $(RM) -r lib; mv lib_ lib; fi
 	$(RM) -r node_modules
 	$(RM) -r gcovr build
-	$(RM) -r out/$(BUILDTYPE)/.coverage
-	$(RM) out/$(BUILDTYPE)/obj.target/node/gen/*.gcda
-	$(RM) out/$(BUILDTYPE)/obj.target/node/src/*.gcda
-	$(RM) out/$(BUILDTYPE)/obj.target/node/src/tracing/*.gcda
-	$(RM) out/$(BUILDTYPE)/obj.target/node/gen/*.gcno
-	$(RM) out/$(BUILDTYPE)/obj.target/node/src/*.gcno
-	$(RM) out/$(BUILDTYPE)/obj.target/node/src/tracing/*.gcno
-	$(RM) out/$(BUILDTYPE)/obj.target/cctest/src/*.gcno
-	$(RM) out/$(BUILDTYPE)/obj.target/cctest/test/cctest/*.gcno
-	$(RM) out/$(BUILDTYPE)/obj.target/embedtest/src/*.gcno
-	$(RM) out/$(BUILDTYPE)/obj.target/embedtest/test/embedding/*.gcno
+	$(RM) -r coverage/tmp
+	$(FIND) out/$(BUILDTYPE)/obj.target \( -name "*.gcda" -o -name "*.gcno" \) \
+		-exec $(RM) {} \;
```

Optional suggestion: `-type f` for safety unless you expect symlinks or something?

```suggestion
	$(FIND) out/$(BUILDTYPE)/obj.target \( -name "*.gcda" -o -name "*.gcno" \) \
		-type f -exec $(RM) {} \;
```